.github/workflows/phpunit-tests.yml

```diff
@@ -224,12 +224,12 @@ jobs:
       report: ${{ false }}
 
   #
-  # Runs specific individual test groups.
+  # Runs HTML API test groups.
   #
   # These tests are run against the most recent LTS version of MySQL.
```

Comment:
Let's add an additional sentence as to why these are tested separately while we're at it.